src/phantom.js

```diff
@@ -18,18 +18,15 @@ var phantom;
  * @param  {Object}  instance   The instance to use, if there is one
  * @return {promise}
  */
-function init(instance) {
+function init(instance, options) {
     if (instance) {
         phantom = instance;
         return null;
     }
 
     // Convert to bluebird promise
     return new promise(function (resolve) {
-        resolve(phridge.spawn({
-            ignoreSslErrors: 'yes',
-            sslProtocol: 'any'
-        }));
```

Comment:
I'm not a maintainer, but you should keep these settings as defaults. You can do this with:

```
options = _.assign({
            ignoreSslErrors: 'yes',		
            sslProtocol: 'any'
}, options);
```